src/ripple/app/tx/impl/details/NFTokenUtils.cpp

```diff
@@ -520,34 +520,55 @@ findTokenAndPage(
     }
     return std::nullopt;
 }
-void
-removeAllTokenOffers(ApplyView& view, Keylet const& directory)
-{
-    view.dirDelete(directory, [&view](uint256 const& id) {
-        auto offer = view.peek(Keylet{ltNFTOKEN_OFFER, id});
 
-        if (!offer)
-            Throw<std::runtime_error>(
-                "Offer " + to_string(id) + " not found in ledger!");
+int
```

Comment:
Nitpick. ```int```'s size is implementation-dependent. Ideally, we want to control the data size (e.g. ```uint64_t```).

Reply:
would `size_t` work?

Reply:
Yes. `size_t` is implementation-dependent but it can store the maximum size of any type (including array).

An STL container's functions `size()` returns a `size_t` result.

https://en.cppreference.com/w/cpp/types/size_t